### DISCUSSION TITLE
[Mobile] how to clean content in the QuillEditor ?

### DISCUSSION THREAD

**Discussion by reamal** [singerdmx/flutter-quill#503] [General]
My issue is about [Mobile]
I try clean content in the QuillEditor.

**Comment by reamal** [comment]
change _controller value is work, but should have a better way ?
`setState(() {
      _controller = QuillController.basic();
    });`

**Comment by MagTuxGit** [comment]
``` dart
  void clean() {
    replaceText(0, plainTextEditingValue.text.length-1, '', TextSelection.collapsed(offset: 0));
  }
```

**Reply by MagTuxGit** [reply]
Implemented here
https://github.com/singerdmx/flutter-quill/pull/505
